src/gdcdictionary/schemas/exposure.yaml

```diff
@@ -294,6 +294,12 @@ properties:
         $ref:
           - "_terms_enum.yaml#/not_reported/common"
 
+  exposure_duration_hrs_per_day:
+    description: >-
+      The duration (in hours per day) that a person was exposed.
+    type: number
+    minimum: 0
```

Comment:
this isn't in the ticket, but is a max of 24 reasonable?

Reply:
Good point, added to ticket and yaml.